API/Controllers/BookController.cs

```diff
@@ -153,7 +153,9 @@ public async Task<ActionResult<string>> GetBookPage(int chapterId, [FromQuery] i
         if (chapter == null) return BadRequest(await _localizationService.Translate(User.GetUserId(), "chapter-doesnt-exist"));
         var path = _cacheService.GetCachedFile(chapter);
 
-        var baseUrl = "//" + Request.Host + Request.PathBase + "/api/";
+        string basePath = Configuration.BaseUrl;
```

Comment:
Please stick to coding style of the codebase. We use var and string.Empty. 

This line here is not needed. You would rewrite this as:
`var basepath = Configuration.BaseUrl == Configuration.DefaultBaseUrl ? string.Empty : Configuration.BaseUrl;`

Reply:
Done :)